src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+
+    private static final String divider = "____________________________________________________________";
+
+    private static final String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Displays a greeting message.
+     */
+    public void greet() {
+        System.out.println("Hello from\n" + logo +"\n");
+        System.out.println("What can I do for you?");
+        showLine();
+    }
+
+    /**
+     * Displays a goodbye message.
+     */
+    public void goodBye() {
+        scanner.close();
+        System.out.println("Bye. Hope to see you again soon!");
+        showLine();
+    }
```

Comment:
Maybe the method names could be phrased as an action ie. (printGreet / printGoodbye) ?